Formula/luit.rb

```diff
@@ -4,6 +4,11 @@ class Luit < Formula
   url "https://invisible-mirror.net/archives/luit/luit-20190106.tgz"
   sha256 "2b900f65ccdc38f8bfc11c6020069d055ba63fce6f90baefe8efc222a5ca3920"
 
+  livecheck do
+    url "https://invisible-mirror.net/archives/luit/"
+    regex(/href=.*?luit[._-]v?(\d+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?luit[._-]v?(\d+(?:[.-]\d+)*)\.t/i)
```

This suggested regex would also match versions like `YYYY-MM-DD` or something like `1.2.3` if the format completely changed.

The alternative is to simply be more explicit about this being a date, like `v?(\d{2,4}-?\d{2}-?\d{2})` (which would also optionally match `YYYY-MM-DD`, `MMDDYY`, `MM-DD-YY`, etc.). In the past, I've usually tried to make it apparent in the regex when a version is a date, by specifying the number of digits involved, but I'm fine with the more open-ended regex suggested above.

Reply:
adopt the changes.